docs/sources/panels/transform-data/transformation-functions.md

```diff
@@ -245,6 +245,37 @@ We would then get :
 
 This transformation allows you to extract some key information out of your time series and display them in a convenient way.
 
+## Grouping to matrix
```

Comment:
This looks great, can you add a copy of this docs to the in-app docs for this transform? 
Example: 
https://github.com/grafana/grafana/blob/main/public/app/features/transformers/configFromQuery/ConfigFromQueryTransformerEditor.tsx#L96